spacy/lang/ti/lex_attrs.py

```diff
@@ -11,67 +11,47 @@
     "ሸውዓተ",
     "ሽሞንተ",
     "ትሽዓተ",
-    "ኣሰርተ",
-    "ኣሰርተ ሐደ",
-    "ኣሰርተ ክልተ",
-    "ኣሰርተ ሰለስተ",
-    "ኣሰርተ ኣርባዕተ",
-    "ኣሰርተ ሓሙሽተ",
-    "ኣሰርተ ሽድሽተ",
-    "ኣሰርተ ሸውዓተ",
-    "ኣሰርተ ሽሞንተ",
-    "ኣሰርተ ትሽዓተ",
+    "ዓሰርተ",
+    "ዓሰርተ ሓደ",
+    "ዓሰርተ ክልተ",
+    "ዓሰርተ ሰለስተ",
+    "ዓሰርተ ኣርባዕተ",
+    "ዓሰርተ ሓሙሽተ",
+    "ዓሰርተ ሽድሽተ",
+    "ዓሰርተ ሸውዓተ",
+    "ዓሰርተ ሽሞንተ",
+    "ዓሰርተ ትሽዓተ",
```

Comment:
I think that these entries were a mistake in this list because the tokenizer isn't going to produce tokens with spaces in them, so these are never going to match. It looks like each of the individual tokens is already in the list, so I don't think we need the additional entries with space at all ... or am I misunderstanding the details here?

Reply:
That's a good point, @adrianeboyd. I wondered about that myself but again since they existed in the previous version, I left that in tact. And the tokens do exist separately in the list, so unless there is a criteria where we need to cover all numbers sequentially, we can remove of the double word entries. I will push a commit with the changes.

Reply:
Done.

BTW, similar issues exist with Amharic [am] as well. We can have those changes as part of this PR if you want.

Reply:
If you don't mind, I think a separate PR would be cleaner. (Create a new branch first! It's better not to create PRs directly from `master` because if you do and then you update or modigy `master` in your repo, it will affect the PR here, too.)